File: 06 - Java/09-01-00-LogitudCadena-ULP.java
public class ProgramaPrincipal {

    public static void main(String[] args){
        // Obtener el largo de una cadena
        var cadena = "Hola Mundo";
        System.out.println("Largo cadena: " + cadena.length());

        // Recorrer la cadena
        for (int i = 0; i < cadena.length(); i++){
            System.out.println(i + " - " + cadena.charAt(i));
        }
    }
}

File: 05 - Python/09-01-00-LogitudCadena-ULP.py
# Funciones incorporadas en Python

# Obtener el largo de una cadena
cadena = 'Hola'
print(f'Largo cadena: {len(cadena)}')

print(cadena[0])
#cadena[0] = 'A'

cadena = 'Adios'
print(cadena)

# Recorrer los elementos de una cadena
for indice, caracter in enumerate(cadena):
    print(f'{indice} - {cadena[indice]}')
    #print(f'{indice} - {caracter}')
